modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/configuration/DistributionZoneConfigurationSchema.java

```diff
@@ -53,6 +56,11 @@ public class DistributionZoneConfigurationSchema {
     @Value(hasDefault = true)
     public int replicas = DEFAULT_REPLICA_COUNT;
 
+    /** Data storage configuration. */
+    @KnownDataStorage
```

Comment:
UnknownDataStorageConfigurationSchema is still there, also some obsolete (I think) machinery around it. But the scope of the current issue is a big enough already. Can I create the separate issue for removing the whole UnknownDataStorageConfigurationSchema from all tests and places? I can put it in the nearest sprint, I think, wdyt?

Reply:
But what about this validator - removed

Reply:
If we stop using the "unknown" storage in current issue, I believe it would also be a right place to to remove everything related to it, like schemas, tests and validators.

Reply:
Done